data/en/getfilefrompath.json

```diff
@@ -14,5 +14,14 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/getfilefrompath"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/getfilefrompath"}
 	},
-	"links": []
+	"links": [],
+	"examples" : [
+		{
+			"title":"Basic Example",
+			"description":"",
+			"code":"fileName = getFileFromPath(\"C:\\temp\\file.txt\"));",
+			"result":"The fileName variable will contain the string file.txt",
+			"runnable":false
```

Comment:
If the above changes are made this can be set runnable to `true`

Reply:
Got it, thanks @pfreitag I get how this works now.